packages/pigeon/platform_tests/shared_test_plugin_code/lib/integration_tests.dart

```diff
@@ -299,6 +307,15 @@ void runPigeonIntegrationTests(TargetGenerator targetGenerator) {
       expect(receivedInt, sentInt);
     });
 
+    testWidgets('Int64 serialize and deserialize correctly',
+        (WidgetTester _) async {
+      final HostIntegrationCoreApi api = HostIntegrationCoreApi();
+
+      const int sentInt = 3000000000;
```

Comment:
Could you make a file-level private constant for this (e.g., `_veryLargeIntValue`), instead of repeating the magic number in a bunch of places? It can have a comment explaining that it's a value chosen to be outside of the 32-bit integer range.

Reply:
of course